alerting/provider/pushover/pushover.go

```diff
@@ -48,6 +48,11 @@ type Config struct {
 	// Sound of the messages (see: https://pushover.net/api#sounds)
 	// default: "" (pushover)
 	Sound string `yaml:"sound,omitempty"`
+
+	// Time to Live of your message (https://pushover.net/api#ttl)
```

Comment:
```suggestion
	// TTL of your message (https://pushover.net/api#ttl)
```
First word needs to match the name of the variable/field